packages/SketchMorph2-Core.package/M2Canvas.class/instance/mouseEnterDragging..st

```diff
@@ -1,4 +1,4 @@
 event handling
 mouseEnterDragging: evt
-	self backend isInGeometricMode ifTrue: [^ nil].
-	self backend startDrawing
\ No newline at end of file
+	self backend context isInGeometricMode ifTrue: [^ nil].
```

Comment:
I implemented it as you described for now. It would be even better if we could get rid of all method calls to `M2DrawingContext` in `M2Canvas` in the future (it's only being called twice).